render/jpeg2k/jpeg2k_encoder_cgo.go

```diff
@@ -0,0 +1,190 @@
+/*
```

Comment:
this file name does not need to have _cgo.go ?
Since this is just a supporting package, not an example